build/templates/session.py.mako

```diff
@@ -195,12 +195,12 @@ constructor_params = helper.filter_parameters(init_function, helper.ParameterUsa
         self._encoding = encoding
 
         # Store the parameter list for later printing in __repr__
-        param_list = []
-        param_list.append("repeated_capability_list=" + pp.pformat(repeated_capability_list))
-        param_list.append("${config['session_handle_parameter_name']}=" + pp.pformat(${config['session_handle_parameter_name']}))
-        param_list.append("library=" + pp.pformat(library))
-        param_list.append("encoding=" + pp.pformat(encoding))
-        self._param_list = ', '.join(param_list)
+        self._params = {
+            'repeated_capability_list': repeated_capability_list,
+            '${config['session_handle_parameter_name']}': ${config['session_handle_parameter_name']},
+            'library': library,
+            'encoding': encoding,
+        }
```

Comment:
Note that `freeze_it` is not getting stored. Is this by design? Looks like a bug.